src/components/UI/Overlay/OverlayContent/AddressBook.tsx

```diff
@@ -76,6 +78,16 @@ export const AddressBook = styled(({ onAddressSelected, ...props }: AddressBookP
     }
   };
 
+  const queryFeatures = async () => {
+    try {
+      const { features } = await getFeatures(endpoint, apiHeader, apiKey);
+      setEntityLookupPath(features.entityLookup?.location);
+    } catch (e) {
+      setEntityLookupPath(undefined);
+      console.log(e, "error");
+    }
+  };
+
```

Comment:
This shouldn't be done here. If not every search term will fire one query feature. 

![image](https://user-images.githubusercontent.com/7406870/96424373-35c71580-122d-11eb-90e7-92b3d5ccc61c.png)

![image](https://user-images.githubusercontent.com/7406870/96424480-54c5a780-122d-11eb-893c-63ce1dba185b.png)

In fact this information is already available at `useThirdPartyAPIEndpoints` hook already. To make it even more explicit, can export a function from there `hasFeature(feature: "ADDRESS_RESOLUTION" | "ENTITY_LOOKUP", index:number)`

Reply:
ah yes, didn't realise it's there. ok thanks will update